sdks/python/apache_beam/transforms/external.py

```diff
@@ -414,7 +414,7 @@ def __init__(
 
   def expand(self, pcolls):
     # Expand the transform using the expansion service.
-    return pcolls | ExternalTransform(
+    return pcolls | self._payload_builder._identifier >> ExternalTransform(
```

Comment:
Let's not use private variable "_identifier" here. (either use something else or add a public property.